OpenRA.Game/Activities/Activity.cs

```diff
@@ -265,6 +281,11 @@ public virtual IEnumerable<Target> GetTargets(Actor self)
 		{
 			yield break;
 		}
+
+		public virtual TargetLineNode? TargetLineNode(Actor self)
+		{
+			return null;
```

Comment:
The comment above explains to create a `TargetLineNode` with `Target.Invalid` for non-drawables, which conflicts with returning null here?

Reply:
But an even better option (which i've brought up on earlier PRs) is to make this `public virtual IEnumerable<TargetLineNode> TargetLineNodes(Actor self)` to match `GetTargets` and allow `yield break`ing on activities that don't want to return anything.